main.py

```diff
@@ -1,10 +1,49 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as date:
+        player_info = json.load(date)
+    for player, info in player_info.items():
+        race_name = info["race"]["name"]
+        skills = info["race"]["skills"]
+        guild = info["guild"] if info["guild"] else None
+
+        if not Race.objects.filter(name=race_name).exists():
```

Comment:
You can use `get_or_create` if objects don't exit https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create
here and other places